SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKDefaultLaunchConfigGenerator.h"
+
+#include "SpatialGDKEditorSettings.h"
+
+#include "Serialization/JsonWriter.h"
+#include "Misc/FileHelper.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialGDKDefaultLaunchConfigGenerator);
+
+#define LOCTEXT_NAMESPACE "SpatialGDKDefaultLaunchConfigGenerator"
+
+using namespace SpatialGDK;
+
+namespace
+{
+bool WriteFlagSection(TSharedRef<TJsonWriter<>> Writer, const FString& Key, const FString& Value)
+{
+	Writer->WriteObjectStart();
+		Writer->WriteValue(TEXT("name"), Key);
+		Writer->WriteValue(TEXT("value"), Value);
+	Writer->WriteObjectEnd();
+
+	return true;
+}
+
+bool WriteWorkerSection(TSharedRef< TJsonWriter<> > Writer, const FWorkerTypeLaunchSection& Worker)
+{
+	Writer->WriteObjectStart();
+		Writer->WriteValue(TEXT("worker_type"), *Worker.WorkerTypeName.ToString());
+		Writer->WriteArrayStart(TEXT("flags"));
+		for (const auto& Flag : Worker.Flags)
```

Comment:
Nit: I think this loop should be indented to match what happens elsewhere

Reply:
Updated.